uber/custom_tags.py

```diff
@@ -423,8 +423,7 @@ def normalize_newlines(text):
 
 @JinjaEnv.jinja_export
 def csrf_token():
-    if not cherrypy.session.get('csrf_token'):
-        cherrypy.session['csrf_token'] = uuid4().hex
```

Comment:
the original code was actually doing the right thing here, and was refactored into ensure_csrf_token_exists()